customer-models/src/main/java/org/powertac/customer/evcharger/DemandSampler.java

```diff
@@ -46,10 +48,18 @@
   private static final Logger log = LogManager.getLogger(DemandSampler.class.getSimpleName());
   private MixtureMultivariateNormalDistribution pluginProbability;
   private HashMap<String, MixtureMultivariateNormalDistribution> condHorizonDemandProbabilities = new HashMap<>();
-  private Integer seed;
+  private Long currentSeed;
+  private RandomDataGenerator randomSeedGenerator;
   private XMLConfiguration config;
   private boolean enabled = true;
 
+  void initialize (String model, RandomSeed randomSeed)
+  {
+    this.initialize(model);
+    randomSeedGenerator = new RandomDataGenerator();
+    randomSeedGenerator.reSeed(randomSeed.getValue());
```

Comment:
This is where the randomSeed is used. I forgot to add this line to the other comment ([#r897885470](https://github.com/powertac/powertac-server/pull/1134/files/18327a2cdda64ded2411688aa9ac3f28add31bd2#r897885470)).